code/modules/vehicles/vehicle_actions.dm

```diff
@@ -103,10 +103,53 @@
 /datum/action/vehicle/sealed/climb_out
 	name = "Climb Out"
 	desc = "Climb out of your vehicle!"
+	button_icon_state = "car_eject"
 
 /datum/action/vehicle/sealed/climb_out/Trigger()
 	if(..() && istype(vehicle_entered_target))
 		vehicle_entered_target.mob_try_exit(owner, owner)
 
 /datum/action/vehicle/ridden
 	var/obj/vehicle/ridden/vehicle_ridden_target
+
+/datum/action/vehicle/sealed/remove_key
+	name = "Remove key"
+	desc = "Take your key out of the vehicle's ignition"
+	button_icon_state = "car_removekey"
+
+/datum/action/vehicle/sealed/remove_key/Trigger()
+	vehicle_entered_target.remove_key(owner)
+
+//CLOWN CAR ACTION DATUMS
+/datum/action/vehicle/sealed/horn
+	name = "Honk Horn"
+	desc = "Honk your classy horn."
+	button_icon_state = "car_horn"
+	var/hornsound = 'sound/items/carhorn.ogg'
+	var/last_honk_time
+
+/datum/action/vehicle/sealed/horn/Trigger()
+	if(world.time - last_honk_time > 20)

```

Comment:
imo this should have the cooldown be a variable and have these two procs not be so duplicated but hey whatever